docs/pom.xml

```diff
@@ -17,8 +17,8 @@
     <properties>
         <!-- The Graal version we suggest using in documentation - as that's
            what we work with by self downloading it: -->
-        <graal-sdk.version-for-documentation>22.3</graal-sdk.version-for-documentation>
```

Comment:
Is this really necessary to change everywhere ? I'm here thinking of handling updates/backports of docs. 

Maybe not a super big deal but just wondering why change this perfectly valid key?

Reply:
Because we use that variable to indicate where to download the GraalVM Community packages, and we want to recommend jdk-17.0.7 (aka 23.0)